starfish/test/experiment/test_experiment.py

```diff
@@ -1,17 +1,93 @@
+from typing import Tuple
+
+import numpy as np
+from slicedimage import Tile, TileSet
+
 import starfish.data
 from starfish.experiment.experiment import Experiment, FieldOfView
-from starfish.imagestack.imagestack import ImageStack
-from starfish.imagestack.parser.crop import CropParameters
+from starfish.types import Axes, Coordinates
 from starfish.util.synthesize import SyntheticData
 
 
+def round_to_x(r: int) -> Tuple[float, float]:
+    return (r + 1) * 1000, (r + 1) * 100
+
+
+def round_to_y(r: int) -> Tuple[float, float]:
+    return (r + 1) * 10, (r + 1) * 0.1
+
+
+def round_to_z(r: int) -> Tuple[float, float]:
+    return (r + 1) * 0.01, (r + 1) * 0.001
+
+
+NUM_ROUND = 5
+NUM_CH = 2
+NUM_Z = 1
+HEIGHT = 10
+WIDTH = 10
+
+
+def get_aligned_tileset():
+    alignedTileset = TileSet(
+        [Axes.X, Axes.Y, Axes.CH, Axes.ZPLANE, Axes.ROUND],
+        {Axes.CH: NUM_CH, Axes.ROUND: NUM_ROUND, Axes.ZPLANE: NUM_Z},
+        (HEIGHT, WIDTH))
+
+    for r in range(NUM_ROUND):
+        for ch in range(NUM_CH):
+            for z in range(NUM_Z):
+                tile = Tile(
+                    {
+                        Coordinates.X: 1,
+                        Coordinates.Y: 4,
+                        Coordinates.Z: 3,
+                    },
+                    {
+                        Axes.ROUND: r,
+                        Axes.CH: ch,
+                        Axes.ZPLANE: z,
+                    }
+                )
+                tile.numpy_array = np.zeros((100, 100))
+                alignedTileset.add_tile(tile)
+    return alignedTileset
+
+
+def get_un_aligned_tileset():
```

Comment:
do we want to test the case where we have multiple tiles that are aligned, yet not all tiles are?

Reply:
added!